pkgs/tools/security/ssh-mitm/default.nix

```
@@ -1,55 +1,43 @@
-{ lib
-, fetchFromGitHub
-, python3
+{ lib,
+fetchFromGitHub,
+python3,
+stdenv
 }:
```

```suggestion
{
  lib,
  stdenv,
  fetchFromGitHub,
  python3,
}:
```

to follow the latest RFC about formatting.